src/__swingset-components/swingset-test-icon/index.tsx

```diff
@@ -1,16 +1,18 @@
 import { ReactElement } from 'react'
```

Comment:
Added home-16 and home-24 icons here, alphabetized the imports, and updated the `SwingsetTestIconProps` `name` prop to reflect all icon options.

Reply:
Removed, no longer relevant! ✨